src/class-parsely.php

```diff
@@ -1107,4 +1114,23 @@ public function get_api_key(): string {
 
 		return $this->api_key_is_set() ? $options['apikey'] : '';
 	}
+
+	/**
+	 * Get the default category for posts.
+	 *
+	 * @since 3.2.0
+	 *
+	 * @return WP_Term|null The category, or null if a category couldn't be retrieved.
+	 */
+	public function get_default_category(): ?WP_Term {
+		$result      = null;
+		$category_id = get_option( 'default_category' );
+		$categories  = get_categories( array( 'ID' => $category_id ) );
```

Comment:
Why using `get_categories()` with a single ID, and then the first from the resulting array? Why not `get_category()` (singular) with the ID?